### PR TITLE
improving storage files in new branch

### PR DESCRIPTION
**Pull request by ewurapoms**
creating pull request for new branch

### CODE REVIEW

**Comment by KafuiPraise** [comment]
Merging pull request

**Review by ewurapoms — COMMENTED** [review]
Changes reviewed by Pomaa.

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -51,6 +51,28 @@ def all(self, cls=None):
                     new_dict[key] = obj
         return (new_dict)
 
+    def get(self, cls, id):
```
Comment (Dedeyd70) [inline models/engine/db_storage.py:54]: great work

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -34,6 +34,30 @@ def all(self, cls=None):
             return new_dict
         return self.__objects
 
+    def get(self, cls, id):
```
Comment (Dedeyd70) [inline models/engine/file_storage.py:37]: merging

**Review by Dedeyd70 — COMMENTED** [review]
merged files. great work

**Pull request merged**